hammers/query.py

```diff
@@ -284,6 +284,52 @@ def update_orphan_resource_providers(db):
     '''
     return db.query(sql, no_rows=True)
 
+@query
+def get_advanced_reservations(db):
```

Comment:
It should be advance reservations, not advanced. Please replace all occurrences in function names and comments.

Reply:
Not completely fixed, see https://github.com/ChameleonCloud/hammers/pull/3/files#r215293123